src/components/pipelines-metrics/PipelinesMetricsPage.tsx

```diff
@@ -51,6 +40,8 @@ const PipelinesMetricsPage: React.FC = () => {
           domain={{ y: [0, 100] }}
           bordered={false}
           namespace={namespace}
+          parentName={parentName}
```

Comment:
We need a check when a user directly changes the namespace from the dropdown on the Metrics Tab - it should redirect to the Pipelines tab or show `No metrics found`. Currently gives error on switching. 
The top left corner drop down.

![project-drop](https://github.com/openshift-pipelines/console-plugin/assets/17771352/508a7b7d-3a56-498f-bc26-224c571c46c3)

Reply:
Error message -
![error](https://github.com/openshift-pipelines/console-plugin/assets/17771352/65aa2aae-7872-4aa3-b34e-518ec8440716)

Reply:
It is a console issue that needs to fix it in the console repository.